### PR TITLE
GitHub extension - overall edits, 2 new screenshots, description

### PR DESCRIPTION
**Pull request by Catografix**
overall edits, 
2 new screenshots, 
1 recropped image
description

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

File: articles/extensions/github-deploy.md [hunk 1]
```diff
@@ -1,52 +1,61 @@
...
 
-At this point you must set the following configuration variables:
+Set the following configuration variables:
 
-- **GITHUB_REPOSITORY**: The repository from which you want to deploy rules and database scripts. It can be either public or private. 
+- **GITHUB_REPOSITORY**: The repository from which you want to deploy rules and database scripts. This can be either a public or private repository. 
 - **GITHUB_BRANCH**: The branch that the extension will monitor for commits.
-- **GITHUB_TOKEN**: Your [personal access token](https://help.github.com/articles/creating-an-access-token-for-command-line-use/#creating-a-token) for GitHub.
-- **SLACK_INCOMING_WEBHOOK_URL**: The Webhook URL for Slack, used in order to get Slack notifications on successful and failed deployments. Setting this value is optional.
+- **GITHUB_TOKEN**: Your GitHub personal access token. Follow the instructions at [Creating an access token](https://help.github.com/articles/creating-an-access-token-for-command-line-use/#creating-a-token) to create a token with `admin:repo_hook` scope.
```
Comment (Catografix) [inline articles/extensions/github-deploy.md]: @mpaktiti I added scope requirement, is this correct?

File: articles/extensions/github-deploy.md [hunk 2]
```diff
@@ -1,52 +1,61 @@
...
-Click on the row displayed for __GitHub Deployments__ extension. The first time you click on your installed extension you will be asked to authorize it to access your GitHub account. Once you agree you will be navigated to the __GitHub Integration__ configuration page.
+Once you agree, you will be directed to the __GitHub Integration__ configuration page.
 
 ![](/media/articles/extensions/github-deploy/configure-extension.png)
 
-The first page is the __Configuration__ and it will explain how you can create a [webhook](https://help.github.com/articles/about-webhooks/) in your GitHub repository pointing to the extension. 
+The __Configuration__ page will display the settings you will need to create a [webhook](https://developer.github.com/webhooks/) in your GitHub repository pointing to the extension. 
 
-> You can find details on how to configure a webhook in [GitHub documentation](https://developer.github.com/webhooks/creating/).
+Copy these values into the **Add Webhook** page for your GitHub repository:
```
Comment (Catografix) [inline articles/extensions/github-deploy.md:38]: I added this info, the webhook was not created automatically.

**Comment by Catografix** [comment]
Preview OK: https://auth0-docs-content-pr-1906.herokuapp.com/docs/extensions/github-deploy

File: articles/extensions/github-deploy.md (hunk 1 above)
Comment (mpaktiti) [inline articles/extensions/github-deploy.md]: @Catografix In my tests I didn't have to add this scope. I had only these set:

---

image

**Comment by Catografix** [comment]
@mpaktiti Made that change

**Comment by Catografix** [comment]
Preview OK: https://auth0-docs-content-pr-1906.herokuapp.com/docs/extensions/github-deploy

**Pull request merged**